integreat_cms/cms/templates/users/region_user_list_row.html

```diff
@@ -6,43 +6,28 @@
             {{ user.username }}
         </a>
     </td>
-	<td>
-        <a href="{% url 'edit_region_user' region_slug=request.region.slug user_id=user.id %}" class="block py-3 pr-2 text-gray-800">
-            {{ user.first_name }}
-        </a>
+	<td class="pr-2">
+		{{ user.first_name }}
 	</td>
-	<td>
-        <a href="{% url 'edit_region_user' region_slug=request.region.slug user_id=user.id %}" class="block py-3 pr-2 text-gray-800">
-            {{ user.last_name }}
-        </a>
+	<td class="pr-2">
+		{{ user.last_name }}
 	</td>
-	<td>
+	<td class="pr-2">
 		<span class="hover:underline text-blue-500 hover:text-blue-600">
             {{ user.email|urlize }}
 		</span>
 	</td>
-	<td>
-        <a href="{% url 'edit_region_user' region_slug=request.region.slug user_id=user.id %}" class="block py-3 pr-2 text-gray-800">
-			{% if user.organization %}
-				{{ user.organization }}
-			{% endif %}
-        </a>
-	</td>
-	<td>
-		<a href="{% url 'edit_region_user' region_slug=request.region.slug user_id=user.id %}" class="block py-3 pr-2 text-gray-800">
-			{{ user.role }}
-		</a>
+	<td class="pr-2">
+		{{ user.role }}
 	</td>
-	<td>
-		<a href="{% url 'edit_user' user_id=user.id %}" class="block py-3 pr-2 text-gray-800 whitespace-nowrap">
```

Comment:
This faulty url (`edit_user` instead of `edit_region_user`) was leading to a few permission denied errors when the user form was opened by clicking on the login date

Reply:
Good catch